productcomments.php

```diff
@@ -929,6 +929,7 @@ public function hookDisplayFooterProduct($params)
      */
     public function hookFilterProductContent(array $params)
     {
+        require_once dirname(__FILE__) . '/ProductComment.php';
```

Comment:
It's strange you need to do this. Is this line necessary when you use the module as a widget?

Reply:
Yep. As far as I remember when I do the change. Also, I copy it from other methods in the same file.

Reply:
I think you should not require this file, the autoload system should be doing it for you.